pkg/services/sqlstore/stats.go

```diff
@@ -105,6 +105,9 @@ func (ss *SQLStore) GetSystemStats(ctx context.Context, query *models.GetSystemS
 		sb.Write(`(SELECT COUNT(*) FROM ` + dialect.Quote("data_keys") + `) AS data_keys,`)
 		sb.Write(`(SELECT COUNT(*) FROM ` + dialect.Quote("data_keys") + `WHERE active = true) AS active_data_keys,`)
 
+		// TODO: table name will change and filter should check only for is_enabled = true
+		sb.Write(`(SELECT COUNT(*) FROM ` + dialect.Quote("dashboard_public_config") + `WHERE 1 = 1) AS public_dashboards,`)
```

Comment:
Whats the point of `WHERE 1=1`

Reply:
As a reminder, this PR is not ready to merge, I need to make changes in this line and remove the comment above after the final DB schema definition.